### PR TITLE
Getting mails

### PR DESCRIPTION
**Pull request by local-minimum**
This does 1. and 2. in the README.md

I have tried running the Pop3 getter manually, not the Imap.

### CODE REVIEW

File: services/inbox/src/inbox/__init__.py [hunk 1]
```diff
@@ -1 +1,18 @@
...
+import os
+from .mailgateway import get_server
+
+host = os.environ.get("EUCO_MAIL_SERVER_HOST", "")  # Defaults to localhost
+port = os.environ.get("EUCO_MAIL_SERVER_PORT", 143)
+user = os.environ.get("EUCO_MAIL_USER", "noone")
+password = os.environ.get("EUCO_MAIL_PWD", "nopwd")
+
+
+def process_mails(host, port, user, password):
```
Comment (skymandr) [inline services/inbox/src/inbox/__init__.py]: How about type hinting?

Comment (local-minimum) [reply services/inbox/src/inbox/__init__.py]: can fix

**Review by skymandr — APPROVED** [review]
I'm sure that there is a case for at the very least catching some broad POP3/IMAP exception in the servers, but that might be difficult to know before something goes wrong. Might we worth looking in imaplib and poplib to see what exceptions they implement, but I'm not sure what is a good way forward. For now, I think this looks good, though I wonder if maybe we should go for type-hint coverage early on? Some IDEs, like PyCh, has nice support for checking it as you code.

**Comment by local-minimum** [comment]
I'll fix type check and linting before continue here and with the errors, I don't think it does exceptions in several cases but statuses in the retreiving. But I don't know, I don't like the docs on pop3/imap.

File: .github/workflows/pythonapp.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Lint with flake8
+      run: |
+        cd services/inbox
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
```
Comment (skymandr) [inline .github/workflows/pythonapp.yml:26]: 127 - booo! ;)

File: services/inbox/src/inbox/__init__.py [hunk 3]
```diff
@@ -2,16 +2,16 @@
 from .mailgateway import get_server
 
 host = os.environ.get("EUCO_MAIL_SERVER_HOST", "")  # Defaults to localhost
```
Comment (skymandr) [inline services/inbox/src/inbox/__init__.py:4]: Not important, but I like the convention of having global scope variables in ALL CAPS to make the stand out as such. Not sure if that's a PEP8 thing or where it comes from.

**Review by skymandr — APPROVED** [review]
Still looks good to me.

File: .github/workflows/pythonapp.yml (hunk 2 above)
Comment (local-minimum) [reply .github/workflows/pythonapp.yml:26]: That was GitHub recommendation

**Pull request merged**